app-text/multitail/files/multitail-7.0.0-gentoo.patch

```diff
@@ -0,0 +1,32 @@
+Gentoo splits off tinfo from ncurses, so we need to ask pkg-config here what the user has on the system.
+
+Signed-off-by: Pascal Jäger <pascal.jaeger@leimstift.de>
+
+--- a/Makefile
++++ b/Makefile
+@@ -24,10 +24,10 @@ ifeq ($(PLATFORM),Darwin)
+     LDFLAGS+=-lpanel $(NCURSES_LIB) -lutil -lm
+ else
+ ifeq ($(UTF8_SUPPORT),yes)
+-    LDFLAGS+=-lpanelw -lncursesw -lutil -lm
++	LIBS+=$(shell $(PKG_CONFIG) --libs ncursesw panelw) -lutil -lm
+     CPPFLAGS+=-DUTF8_SUPPORT
+ else
+-    LDFLAGS+=-lpanel -lncurses -lutil -lm
++	LIBS+=$(shell $(PKG_CONFIG) --libs ncurses panel) -lutil -lm
+ endif
+ endif
+ 
+@@ -40,10 +40,10 @@ DEPENDS:= $(OBJS:%.o=%.d)
+ all: multitail
+ 
+ multitail: $(OBJS)
+-	$(CC) $(OBJS) $(LDFLAGS) -o multitail
++	$(CC) $(CFLAGS) $(OBJS) $(LDFLAGS) -o multitail $(LIBS)
```

Comment:
Nice! Can I suggest throwing in CPPFLAGS too?
```suggestion
+	$(CC) $(CPPFLAGS) $(CFLAGS) $(OBJS) $(LDFLAGS) -o multitail $(LIBS)
```

Reply:
Oh, also, we must do $(OBJS) _after_ $(LDFLAGS), or --as-needed breaks: https://wiki.gentoo.org/wiki/Project:Quality_Assurance/As-needed.

Reply:
Take a close look at that patch now please. Because I am not quite sure if I did this correct. The ccmultitail target had $(LDFLAGS) two times, so I removed the one that was put after the $(OBJS)